### PR TITLE
manually fetches screenshot if no data was provided

### PR DESCRIPTION
**Pull request by sullenor**
simple solution for https://github.com/webdriverio/webdriverio/issues/1229

### CODE REVIEW

**Comment by sullenor** [comment]
Unfortunately, I can't reproduce localy this error:

```bash
Running "eslint:target" (eslint) task
/home/travis/build/webdriverio/webdriverio/lib/utils/ReporterStats.js
  63:7  error  "HookStats" is defined but never used  no-unused-vars
✖ 1 problem (1 error, 0 warnings)
Warning: Task "eslint:target" failed. Use --force to continue.```

**Comment by christian-bromann** [comment]
No worries, I am aware of this situation, haven't been able to take a look yet though

**Comment by sullenor** [comment]
Okay, thanks

File: lib/webdriverio.js [hunk 1]
```diff
@@ -207,19 +207,21 @@ let WebdriverIO = function (args, modifier) {
...
-        if (typeof err.screenshot === 'string') {
+        if (typeof screenshotData === 'string') {
             let screenshot = new Buffer(err.screenshot, 'base64')
             fs.writeFileSync(filePath, screenshot)
         } else {
             client.next(prototype.saveScreenshot, [filePath], 'saveScreenshot')
+            screenshotData = fs.readFileSync(filePath, 'utf8')
         }
         this.logger.log(`\tSaved screenshot: ${fileName}`)
         this.emit('screenshot', {
```
Comment (gronke) [inline lib/webdriverio.js]: There is no need to emit the `screenshot` event when the client manually takes one (like in line 220)

**Comment by christian-bromann** [comment]
Obsolete, see @gronke comment

**Pull request closed without merging**